test/plugins/features/movies/controller.test.js

```diff
@@ -7,18 +7,30 @@ describe('movie controller', () => {
 
   describe('create', () => {
 
-    it('creates a movie', () => {
-      const payload = { title: 'WALL-E' };
-
+    const assertMovieCreated = function (payload, expTitle) {
       return Controller.create(payload)
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(expTitle);
 
         return new Movie({ id: movie.id }).fetch();
       })
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(expTitle);
       });
+    };
```

Comment:
This is a neat way to keep the code DRY -- nice! Only nit is either adding clarity to the expected variable so that it's `expectedTitle` (or really due to this migration it should be `expectedName` or just keeping it as `title`/`name`.

Reply:
Is the coding standard to not abbreviate variables names? i.e. expectedTitle vs expTitle.

Reply:
We tend to prefer clarity over a couple letters saved, but YMMV.

Reply:
Ok. I'll be more verbose.